### DISCUSSION TITLE
Why "Disable language bar shortcuts"?

### DISCUSSION THREAD

**Discussion by amymor** [Atlas-OS/Atlas#1293] [Development]
Configuration\tweaks\qol\ease-of-access\disable-annoying-features-shortcuts.yml
![image](https://github.com/user-attachments/assets/1df09dce-6622-4df9-a0a3-faad62c8fd1f)

I knew that users can add hotkeys in the language settings, but I think many users don't know where, because it's only accessible through the old options hidden behind the advanced settings...
So these hotkeys are important for multilingual users. And I didn't see any script in the "atlas folder" to override it.
I also don't think it will affect performance or QOL for English-only users, because when you use one language, the hotkey doesn't do anything, right?

**Comment by GenzPN** [comment]
Try this on powershell admin

Set-WinUserLanguageList en-US
